README.md

```diff
@@ -1,2 +1,130 @@
+![diagram showing how to host folder](src/img/storageEventsReadme.png)
+
+4. Copy the link to the index.html file where you have it hosted and insert the url into a schedule. (you can also add this link to the url gadget within a presentation)  
+![diagram showing where to find link to index.html](img/readme-step4.jpg)
+
+##Directions to Modify the Event Theme:##
+
+####Changing the Background Image
+
+The background image for the theme can be changed by locating the following code within the style.css file within the css folder and modifying the path of the url to the image of your choice:
+
+```
+.wrapper:after {
+  ...
+  background-image: url("../img/city.jpeg");
+  ...
+```
+
+
+####Changing the Icon Images
+
+The images used for the icons in the theme are SVGs so that they will scale to fit different resolutions. To change the icons, open index.html, and locate the code for the specific icon within the section that you want to modify. For example, the code for the large calendar icon is found here:
+
+```  
+ <img src="img/calendar2.svg" class="date--calendarIcon" alt="calendar icon">
+```
+
+You can then change the src attribute to the path of the icon of your choice.
+
+
+####Changing The Text Content
+The text content is displayed through the use of the `rise-google-calendar` web component. Full details for this component can be found here # 
+In order for the component to access the data you need to create a public google calendar and ensure that all events you create are also public. The settings menu for the calendar is located in the top right corner of the calendar page. The button to make the event public is found here:
+![screenshot showing button to make event public](src/img/diagramPublicEvent.png)
+
+The next step is to locate the following code in the index.html file:
+
+```
+  <rise-google-calendar
+        id="googleCalendar"
+        calendar-id="cameroncodes.com_l0vl6b8laf08tb39lv2vegd1qg@group.calendar.google.com"
+        start-date="2015-01-01" end-date="2015-12-31">
+        <!-- Example HTML structure -->
+      <!--   <h3>Component Test</h3> -->
+        <table id="calendarTable" class="pure-table pure-table-bordered">
+          <thead>
+          </thead>
+          <tbody>
+          <!-- dynamically populate table with calendar data -->
+          </tbody>
+        </table>
+  </rise-google-calendar>
+
+```
+
+The `calendar-id` attribute should be set to the id for your own public calendar. This id can be found by selecting calendar settings, clicking on the calendars link in the top line, and then clicking on the 'Edit settings' or 'Shared: Edit settings' link corresponding to the calendar you wish to collect the data from. The ID for the calendar can then be copied from here:![screenshot showing calendar ID](src/img/calendarid.png)
+
+The start date and end date can be used to specify a specific range of dates. This theme looks for all upcoming events for the current day. 
```

Comment:
@pcsandford I would remove the mention of start date and end date. These are not set on the instance in `index.html` and start date gets dynamically set in code according to the current day.